media/system/js/share-uncompressed.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @package         Joomla.JavaScript
+ * @copyright       Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license         GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+/**
+ * Calls the sending process of the config class
+ */
+jQuery(document).ready(function ($)
+{
+    $('#sharebutton').click(function ()
+    {
+        var share_draft = {
+            articleId  : $().val(),
+            title: $('jform_title').val(),
+            sharetoken  : $().val(),
```

Comment:
and one more question, does all the fields in our db the data has to be passed for them too? Can you explain adding token in hidden field more please